tests/AutoReview/ProjectCodeTest.php

```diff
@@ -69,7 +69,7 @@ static function ($class) { return !class_exists($class) && !trait_exists($class)
      */
     public function testThatSrcClassHaveTestClass($className)
     {
-        $testClassName = str_replace('PhpCsFixer', 'PhpCsFixer\\Tests', $className).'Test';
+        $testClassName = 'PhpCsFixer\\'.str_replace('PhpCsFixer', 'Tests', $className).'Test';
```

Comment:
looks wrong

Reply:
only a little bit :) it was false positive in the bash command - should I add negative lookahead for this case there?

Reply:
i can discuss lot about "why bash will always give you false positives" for this task, unfortunately :(